app/src/main/kotlin/com/wire/android/ui/calling/SharedCallingViewModel.kt

```diff
@@ -228,9 +236,11 @@ class SharedCallingViewModel @Inject constructor(
 
     fun pauseVideo() {
         viewModelScope.launch {
-            if (callState.isCameraOn) {
-                updateVideoState(conversationId, VideoState.PAUSED)
-                setVideoPreview(conversationId, PlatformView(null))
+            callState.isCameraOn?.let {
+                if (it) {
+                    updateVideoState(conversationId, VideoState.PAUSED)
+                    setVideoPreview(conversationId, PlatformView(null))
+                }
```

Comment:
if the default value is `null` would there be any edge case that it will still remain `null` and this won't happen?

Could have something like:

```suggestion
                if (callState.isCameraOn != null && callState.isCameraOn) {
                    updateVideoState(conversationId, VideoState.PAUSED)
                    setVideoPreview(conversationId, PlatformView(null))
```